src/styles/main.scss

```diff
@@ -1,3 +1,52 @@
+@import "./header.scss";
+@import "./card.scss";
+
 body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  --background-color: #fff;
+  --second-background-color: #00acdc;
+  --text-color: #060b35;
+  --second-text-color: #616070;
+  --border-color: #f3f3f3;
+  --color: #00acdc;
   margin: 0;
 }
+
+.header {
+  text-transform: uppercase;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+
+  &__container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding: 50px 40px;
+    align-self: center;
```

Comment:
would be nice to add some `width` / `max-width` for the container
it looks not good on wider screens:
![image](https://user-images.githubusercontent.com/86923089/226171064-2d3c745e-688f-46ef-97ce-ae9260c8bc06.png)

But since it's not required in the task - not critical and not necessary to fix it